client/src/node.rs

```diff
@@ -111,6 +121,7 @@ impl<DB: Database> Node<DB> {
             .get_account(address, Some(&[slot]), tag)
             .await?;
 
+        let slot = U256::from(slot.as_bytes());
```

Comment:
why do you need this conversion?

Reply:
At a time, `value` would expect a `H256` type, like the function's parameter `slot` type, which is a bug.
In `ethers v2.0.14` they fixed this bug, and now `value` should be of type `U256`, as stated in Ethereum's yellow paper.
So I needed to add this conversion in order for this to work.
Another option would be to change the function's signature, but that would break the api, so I thought it should not be done.